src/components/ListItem.jsx

```diff
@@ -56,6 +66,9 @@ export function ListItem({ listToken, item, itemId }) {
 						onChange={handleChange}
 					/>
 					{name}
+					<span className={`urgency-tag ${purchaseUrgency}`}>
```

Comment:
Such a clever way to include the class name dynamically based on `purchaseUrgency`, I love it!!! 💕